src/error.rs

```diff
@@ -106,3 +106,25 @@ impl error::Error for OsError {}
 impl error::Error for ExternalError {}
 impl error::Error for NotSupportedError {}
 impl error::Error for RunLoopError {}
+
+#[cfg(test)]
+mod tests {
+    #![allow(clippy::redundant_clone)]
+
+    use super::*;
+
+    // Eat attributes for testing
+    #[test]
+    fn attr_coverage() {
```

Comment:
Maybe call this `ensure_fmt_does_not_panic` or similar?

Same for other files.